### DISCUSSION TITLE
DoorBell Service ohne Video für HomePod

### DISCUSSION THREAD

**Discussion by Phil7989** [thkl/hap-homematic#305] [Show and tell]
Hallo zusammen,

benutzt jemand den HomePod als Klingel?
Kann ich das auch mit dem VideoDoorBellAccessory realisieren?
Einfach ein Tastendruck löst die Klingel im HomePod/HomePod mini aus?

Danke & Gruß Philipp

**Comment by gerd2005** [comment]
Das würde mich auch interessieren. 
Du hast vermutlich einen einfachen Klingelknopf an der Tür, und benötigst dann ja noch ein Gerät, welches den Tastendruck erkennt, oder?

**Comment by Phil7989** [comment]
Sorry, ich habe inzwischen den HomeMaticDoorBellAccessory gefunden. Das ist genau das war ich suche. aber so richtig ist der Service noch nicht in HomeKit integriert? 
Gibt es hier vielleicht ein Update dazu?

**Comment by Phil7989** [comment]
> Das würde mich auch interessieren.
> Du hast vermutlich einen einfachen Klingelknopf an der Tür, und benötigst dann ja noch ein Gerät, welches den Tastendruck erkennt, oder?

Man könnte ja auch einen Viertüllen Homematic Taster nehmen um das zu realisieren. oder einen normalen Homematic Taster. Oder man schließt die vorhandene Klingel an ein Homematic IO Modul an

**Comment by gerd2005** [comment]
Wie würdest du das denn über einen virtuellen Taster realisieren? Also was ich meine ist, dass du ja irgendwie noch eine Schnittstelle zwischen dem Tastendruck an der Klingel und dem virtuellen Taster benötigst.

**Comment by Phil7989** [comment]
> Wie würdest du das denn über einen virtuellen Taster realisieren? Also was ich meine ist, dass du ja irgendwie noch eine Schnittstelle zwischen dem Tastendruck an der Klingel und dem betätigen des Tasters benötigst.

Ja das wäre doppelt gemoppelt. Ich habe tatsächlich an meiner Klingel einen Homematic 8-Bit HM-MOD-EM-8Bit und den leitet man dann einfach auf den VideoDoorBellAccessory um bzw. weiter.
Falls das nicht gehen sollte. Dann kann man ja auch ein Programm schreiben. Wenn Klingel geschlossen dann viertueller Tastendruck kurz.

**Comment by gerd2005** [comment]
Ach ok, auf der Suche bin ich ehrlich gesagt die ganze Zeit. Ich habe bislang nur den HmIP-DSD-PCB Gefunden. Mich schrecken jedoch die Batterien ab. Benötigt man bei deinem Batterien?

**Reply by dkrizic** [reply]
Ich habe jahrelang den hier verwendet: 

![](https://www.technikhaus.de/images/product_images/popup_images/1266_0.jpg)

Der wird parallel zur Klingel installiert und "bemerkt" ein Klingeln. Ja, er hat Batterien, die halten aber ewig. Seit ich aber jetzt eine "echte" Netatmo Doorbell habe, ist er arbeitslos.

**Comment by Phil7989** [comment]
> Ach ok, auf der Suche bin ich ehrlich gesagt die ganze Zeit. Ich habe bislang nur den HmIP-DSD-PCB Gefunden. Mich schrecken jedoch die Batterien ab. Benötigt man bei deinem Batterien?

Ich habe bei mir ein Netzteil verbaut.

**Comment by gerd2005** [comment]
Es gibt also kein Modul, welches über die Klingelspannung funktioniert?

**Comment by thkl** [comment]
der DoorBell Service erzeugt ein DoorBell Accessory. Das wird zwar in der HomeApp als nicht unterstützt angezeigt, löst aber Push Meldungen aus, wenn es klingelt.

Der DoorBell Service kann jedem KEY Sensor zugeordnet werden. Also beliebigen Tastern, Virtuellen Tasten, Tastenschnittstellen. Ich benutze den HM-Sen-DB-PCB ..Edit:  ähm der braucht doch Batterien, wie ich gerade gesehen habe, ich habe die aber noch nie getauscht.

**Comment by Phil7989** [comment]
> der DoorBell Service erzeugt ein DoorBell Accessory. Das wird zwar in der HomeApp als nicht unterstützt angezeigt, löst aber Push Meldungen aus, wenn es klingelt.
> 
> Der DoorBell Service kann jedem KEY Sensor zugeordnet werden. Also beliebigen Tastern, Virtuellen Tasten, Tastenschnittstellen. Ich benutze den HM-Sen-DB-PCB ..Edit: ähm der braucht doch Batterien, wie ich gerade gesehen habe, ich habe die aber noch nie getauscht.

Ok, dann werde ich das auf jeden Fall mal mit dem HomePod mini testen.
Ich dachte nur in anderen HomeKit DoorBell Integrationen schon mehr Einstellungsmöglichkeiten gesehen zu haben? (Push)

Vielen Dank für HAP Homematic!

**Comment by gerd2005** [comment]
Hallo,

ich habe nun den HmIP-DSD-PCB an meine Klingel angeschlossen und an der CCU3 angelernt. Zum Test habe ich das Gerät per Direktverknüpfung an ein Rollo gekoppelt. Als ich die Klingel gedrückt habe, ist das Rollo herunter gefahren. Es funktioniert also ;-)

Kann mir jemand erklären, wie es jetzt weiter geht? So sieht es in meiner Home-App aus:

![83247849-C90E-4EEF-8F77-F4885376CE6B](https://user-images.githubusercontent.com/72799068/103458047-a34ef200-4d04-11eb-9068-8546eb90c414.png)

Und so im PlugIn:

![8F75316C-1961-4CB6-839A-93628D9C00B7](https://user-images.githubusercontent.com/72799068/103458258-94693f00-4d06-11eb-8f83-33af865a2b4f.png)

In HomeKit habe ich eingestellt, dass ich eine Push-Benachrichtigung erhalten möchte. Leider funktioniert dass nicht. Es scheint mir so, dass das Signal von dem HmIP-DSD-PCB gar nicht in dem PlugIn ankommt. Woran kann das liegen? Ich habe sicherlich beim Accessory etwas falsch gemacht.

Als Zusatzfrage:
Kann mir jemand sagen, wie der HmIP-DSD-PCB Bei euch in der CCU aussieht?
Bei mir erscheint das Gerät so. Es wird noch nicht einmal ein Status angezeigt. Ist das korrekt?

![FD049132-4197-456E-9123-71499B1C4D07](https://user-images.githubusercontent.com/72799068/103458308-02ae0180-4d07-11eb-91ef-b01859e2de14.png)

**Reply by thkl** [reply]
Es müsste der HomeMaticDoorBellAccessory ausgewählt werden. Der erzeugt eine DoorBell. Leider kennt der HomeMaticDoorBellAccessory  im aktuellen Release noch nicht den Kanaltyp MULTI_MODE_INPUT_TRANSMITTER vom HmIP-DSD-PCB

Ich habe das in https://github.com/thkl/hap-homematic/commit/afd9be4ec762ff446033ebbe6622d365da85f313 hinzugefügt.
ggf das File HomeMaticDoorBellAccessory.js (https://raw.githubusercontent.com/thkl/hap-homematic/master/lib/services/HomeMaticDoorBellAccessory.js ) auf der CCU tauschen

**Comment by gerd2005** [comment]
Also den hier?

![1DEA580C-F23B-4478-A268-CDF6D8160BF5](https://user-images.githubusercontent.com/72799068/103458571-bd3f0380-4d09-11eb-9145-72be97c5d51e.png)

Muss ich dann einen virtuellen Taster anlegen?
![1A57A945-4622-4C37-9D93-E998DE1EC9BE](https://user-images.githubusercontent.com/72799068/103458578-d647b480-4d09-11eb-91e2-4edef6ba782d.png)

**Reply by thkl** [reply]
Nein HomeMaticDoorBellAccessory ... ohne Video im Namen

**Comment by gerd2005** [comment]
Mhh der wird mir nicht angezeigt... Weder in den nornmalen Geräten, noch bei den besonderen Geräten.

![87AF954F-EE83-4FF2-B770-446B8C1A9E5C](https://user-images.githubusercontent.com/72799068/103458627-481ffe00-4d0a-11eb-9a24-8318919142e7.png)
![98A490E8-E781-4A89-8552-BA76E337D61B](https://user-images.githubusercontent.com/72799068/103458631-4bb38500-4d0a-11eb-993d-309f13c8c6ce.png)

**Reply by thkl** [reply]
File ausgetauscht ? Reboot des Services ?

**Comment by gerd2005** [comment]
Ah ok. Das habe ich nicht gecheckt. Gibt es zum Tausch einer Datei eine Anleitung? Also was mir fehlt ist der Weg, wie ich in den „Dateiexplorer“ der CCU komme.

**Reply by thkl** [reply]
das geht zb per FTP Client von einem PC/Mac .. mit einem iPad wirds schwer ;)

**Comment by gerd2005** [comment]
Mhh an der Stelle bin ich wahrscheinlich raus. Da muss ich mich einlesen wie das geht, da ich ja vermutlich in der CCU auch etwas freischalten muss. 

Mit dem kommenden Update wird der DoorBellService aber hinzugefügt? Kannst du schon so ungefähr sagen wann du das kommende Release veröffentlichst?

**Reply by thkl** [reply]
ja da ist er drin. Ich habe noch eine Änderung am RGBAccessory und dann könnte  ich .58 zusammenbauen

**Comment by gerd2005** [comment]
Ok, dann warte ich einfach noch etwas :-) Herzlichen Dank!

**Comment by gerd2005** [comment]
Hallo,

der DoorBellService funktioniert. Auch die HomePods klingeln. Richtig gut.

![44641533-EF75-4E74-BAAE-B387FA6F3A1C](https://user-images.githubusercontent.com/72799068/105077889-75e89f00-5a8d-11eb-8307-7f45fd3ebc81.jpeg)

Einen Verbesserungsvorschlag habe ich dennoch, falls thkl da Einfluss drauf hat.

Beim Klingeln wird in der Pushnachricht der Name des Hauses (Tulpenweg), dann der Raum (Haustür) und dann das Gerät (Türklingel) mitgegeben.

Beim Garagentor ist das anders. Hier wird nur das Haus (Tulpenweg) und dann der Name des Geräts in der Pushbenachrichtigung angezeigt. 

![530E0642-CE4D-47BE-A996-20A91D49F7F7](https://user-images.githubusercontent.com/72799068/105078468-59009b80-5a8e-11eb-9944-c6cc71d8176a.jpeg)

Ist es möglich dieses Verhalten auf den DoorBellService zu übertragen?

**Reply by thkl** [reply]
Ich habe exakt NULL Einfluß auf die Push Meldungen. Diese werden einzig von Homekit selber generiert.
Evtl. könntest Du den Gerätenamen in HAP mal ändern und schauen ob das Auswirkungen hat.

**Comment by meistermolli** [comment]
bei mir war der service leider immer mit deutlichen Verzögerungen.

da ich noch einen Umweg gehen musste.

Hue Smart Button gedrückt, löst Virtuellen CCU Taster im Homekit aus.
Regel in CCU: Wenn Virtueller Klingeltaster gedrückt, dann Klingel auslösen.

bis zur Ccu ging das ganze schnell ab. 

Das auslösen in Homekit des Doorbell Service hat dann aber oft sehr lange gedauert.

Schlimmer wurde das ganze dann, wenn jemand die Klingel z.B. lange oder mehrmals hintereinander gedrückt hatte.


Hat da jemand eine Idee wie ich das weniger fehleranfällig mit den vorhandenen Mitteln lösen könnte?

Nutze das ganze mit der Videodoorbell in Homebridge und da funktioniert das super. Aber leider geht dann auch immer dieses Leere Kamera Fenster auf, wenn jemand klingelt.
